yw_website/yw_website/urls.py

```diff
@@ -32,7 +51,13 @@
     path('home/users/', script_upload_views.users, name = 'users'),
     path('upload/', script_upload_views.model_form_upload, name='upload'),
     #test path- remove later
-    path('home/test', script_upload_views.home, name='test')
+    path('home/test', script_upload_views.home, name='test'),
+
+    #REST API URLS
+    path('api/', include(router.urls)),
```

Comment:
make this route `api/v1/`, see [point #8](https://blog.mwaysolutions.com/2014/06/05/10-best-practices-for-better-restful-api/)